### ISSUE TITLE
Want to change the time between nydus requests to sc2

### ISSUE THREAD

**Issue by SaLemandr** [leigholiver/nydus#4]
Sc2 Scene Switcher is working with big delay (5 secs) as it sending requests to SC2 every 5 secs. Trying to find where to change it, found the Worker on git hub, but in download folder there was no him, how can i change this?

**Comment by leigholiver** [comment]
Hi, do you have any logs from the worker at around the time it is running slowly?
I've just refreshed my memory on the code, and it uses a really simple backoff where it should usually wait 0.5 seconds, and switches to a 5 second timer if it encounters an error.

**Comment by leigholiver** [comment]
I've released a version today with some tweaks to how nydus calculates the delay, I'm hoping that this helps with your issue. Let me know if that is any better :)

**Comment by SaLemandr** [comment]
Yeah, i had a wrong vision of how Worker works. Just wanted to make scene switcher change scenes from game to score screen faster a bit so just put some python commands into exitGame function. Will check your changes and give a feedback about it later :). Thank you for your job from all CIS casters
